README.md

```diff
@@ -104,6 +127,30 @@ Host your own Chocolatey package repository
 
 #### Parameters
 
+##### `chocolatey_server_app_pool_name`
+Set apppool name used by the chocolatey.server website. Defaults to
+'chocolatey.server'.
+
+##### `disable_default_website`
+The default website is stopped to prevent conflicts with the
+chocolatey server website. An alternate port can be defined and
+this option can be set to disabled so it doesn't modify the default
+website. Defaults to 'true'.
+
+##### `packages_folder`
```

Comment:
If I read the code correctly, the server_package_source is the location where only the chocolatey.server nuget package can be found. This setting allows you to remap the c:\tools\chocolatey.server\app_data\packages folder to another location where the actual nuget packages being served are stored. For instance, we store all of our nuget packages in c:\packages. Hope that makes sense.